babel-code-frame/index.d.ts

```diff
@@ -0,0 +1,38 @@
+// Type definitions for babel-code-frame 6.x
+// Project: https://github.com/babel/babel/tree/master/packages
+// Definitions by: Mohsen Azimi <https://github.com/mohsen1>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "babel-code-frame" {
```

Comment:
remove the `declare module "babel-code-frame" {` and leave the `export default ..`  below. the name of the module is inferred from the folder name.